addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -567,34 +567,36 @@ def test_in_invoice_line_onchange_partner_1(self):
             {
                 **self.product_line_vals_1,
                 'partner_id': self.partner_b.id,
+                'account_id': self.product_b.property_account_expense_id.id,
+                'tax_ids': self.tax_purchase_b.ids,
             },
             {
                 **self.product_line_vals_2,
                 'partner_id': self.partner_b.id,
+                'tax_ids': self.tax_purchase_b.ids,
+                'price_total': 184.0,
             },
             {
                 **self.tax_line_vals_1,
                 'partner_id': self.partner_b.id,
-            },
-            {
-                **self.tax_line_vals_2,
-                'partner_id': self.partner_b.id,
+                'name': self.tax_purchase_b.name,
+                'tax_line_id': self.tax_purchase_b.id,
             },
             {
                 **self.term_line_vals_1,
-                'name': 'turlututu installment #1',
+                'name': 'turlututu',
```

Comment:
This change is just wrong.
Don't change a test assertion if you don't know why you are doing it